SPManager/src/artofillusion/spmanager/SPMParameters.java

```diff
@@ -88,7 +89,15 @@ public SPMParameters()
      */
     private void loadPropertiesFile()
     {
-        File f = new File( System.getProperty( "user.home" ), ".spmanagerprefs" );
+        // The old file became obsolete when the server was migrated
+        // HTTPS. Scrub it if it's still around.
+        File oldFile = new File(System.getProperty("user.home"), ".spmanagerprefs");
+        if (oldFile.exists());
```

Comment:
Next to this if will be executed always because of semicolon at line end

Reply:
Good catch. Thanks!